static/js/all-alerts.js

```diff
@@ -131,7 +131,31 @@ let alertColumnDefs = [
     {
         headerName: "State",
         field: "alertState",
-        width:50,
+        width: 50,
+        cellStyle: (params) => {
+            console.log(params.data)
+            const alertState = params.data.alertState
+            console.log(alertState); 
+            let backgroundColor;
+            switch (alertState) {
+                case "Normal":
+                    backgroundColor = "#53DB6E"; // Green background for Normal State
```

Comment:
we should use css file to store the color values. @sonamgupta21 can you check the color palate for this

Reply:
Updated the code according to instruction .